frontend-react/src/pages/data-dashboard/DataDashboardTable/DataDashboardTable.tsx

```diff
@@ -0,0 +1,222 @@
+import Spinner from "../../../components/Spinner";
+import { NoServicesBanner } from "../../../components/alerts/NoServicesAlert";
+import { FilterManager } from "../../../hooks/filters/UseFilterManager";
+import { PaginationProps } from "../../../components/Table/Pagination";
+import Table, {
+    ColumnConfig,
+    TableConfig,
+} from "../../../components/Table/Table";
+import TableFilters from "../../../components/Table/TableFilters";
+import {
+    useOrgDeliveries,
+    DataDashboardAttr,
+} from "../../../hooks/network/DataDashboard/DataDashboardHooks";
+import { RSDelivery } from "../../../config/endpoints/dataDashboard";
+
+import ReceiverServices from "./ReceiverServices";
+
+const extractCursor = (d: RSDelivery) => d.batchReadyAt;
+
+interface DashboardTableContentProps {
+    receivers: RSReceiver[];
+    activeService: RSReceiver | undefined;
+    handleSetActive: (v: string) => void;
+    filterManager: FilterManager;
+    paginationProps?: PaginationProps;
+    isLoading: boolean;
+    receiverList: RSDelivery[] | undefined;
+}
+
+const DashboardTableContent: React.FC<DashboardTableContentProps> = ({
+    receivers,
+    activeService,
+    handleSetActive,
+    filterManager,
+    paginationProps,
+    isLoading,
+    receiverList,
+}) => {
+    const featureEvent = `${FeatureName.DAILY_DATA} | ${EventName.TABLE_FILTER}`;
+
+    const transformDate = (s: string) => {
+        return new Date(s).toLocaleString();
+    };
+    const columns: Array<ColumnConfig> = [
+        {
+            dataAttr: DataDashboardAttr.DATE_SENT,
+            columnHeader: "Date sent to you",
+            sortable: true,
+            transform: transformDate,
+        },
+        {
+            dataAttr: DataDashboardAttr.PROVIDER,
+            columnHeader: "Ordering Provider",
+            feature: {
+                link: true,
+                linkBasePath: "/data-dashboard/facilities-providers/:provider",
+            },
+        },
+        {
+            dataAttr: DataDashboardAttr.FACILITY,
+            columnHeader: "Performing facility",
+            feature: {
+                link: true,
+                linkBasePath: "/data-dashboard/facilities-providers/:facility",
+            },
+        },
+        {
+            dataAttr: DataDashboardAttr.SUBMITTER,
+            columnHeader: "Submitter",
+            feature: {
+                link: true,
+                linkBasePath: "/data-dashboard/facilities-providers/:submitter",
+            },
+        },
+        {
+            dataAttr: DataDashboardAttr.REPORT_ID,
+            columnHeader: "Report ID",
+            feature: {
+                link: true,
+                linkBasePath: "/data-dashboard/report-details/:reportId",
+            },
+        },
+    ];
+
+    const resultsTableConfig: TableConfig = {
+        columns: columns,
+        rows: receiverList || [],
+    };
+
+    if (isLoading) return <Spinner />;
```

Comment:
Should move this to the top of the component so if it's `true`, the rest of the business logic doesn't have to process?